File: Popcorn4/app/src/main/java/com/pandey/popcorn4/db/FvrtMoviesDbObject.java
package com.pandey.popcorn4.db;

import androidx.annotation.Keep;
import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

import com.google.android.gms.common.annotation.KeepName;

@Entity
@Keep
@KeepName
public class FvrtMoviesDbObject {

    @PrimaryKey
    @ColumnInfo(name = "movieId")
    public int movieId;

    @ColumnInfo(name = "moviePoster")
    public String moviePosterPath;

    @ColumnInfo(name = "movieTitle")
    public String movieTitle;

    @ColumnInfo(name = "movieDescription")
    public String movieDescription;

    @ColumnInfo(name = "movieCount")
    public int movieStarCount;

    @ColumnInfo(name = "language")
    public String movieLanguage;

    @ColumnInfo(name = "createdAt")
    public long timestamp;

    public FvrtMoviesDbObject(int movieId,
                              @Nullable String moviePosterPath,
                              @NonNull String movieTitle,
                              @Nullable String movieDescription,
                              int movieStarCount,
                              @Nullable String movieLanguage,
                              long timestamp) {
        this.movieId = movieId;
        this.moviePosterPath = moviePosterPath;
        this.movieDescription = movieDescription;
        this.movieTitle = movieTitle;
        this.movieStarCount = movieStarCount;
        this.movieLanguage = movieLanguage;
        this.timestamp = timestamp;
    }
}

File: Popcorn4/app/src/main/java/com/pandey/popcorn4/data/network/db/FvrtMoviesDbObject.kt
package com.pandey.popcorn4.data.network.db

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "FvrtMoviesDbObject")
data class FvrtMoviesDbObject(

        @PrimaryKey
        @ColumnInfo(name = "movieId")
        var movieId: Int,

        @ColumnInfo(name = "moviePoster")
        var moviePosterPath: String,

        @ColumnInfo(name = "movieTitle")
        var movieTitle: String?,

        @ColumnInfo(name = "movieDescription")
        var movieDescription: String,

        @ColumnInfo(name = "movieCount")
        var movieStarCount: Int,

        @ColumnInfo(name = "language")
        var movieLanguage : String,

        @ColumnInfo(name = "createdAt")
        var timestamp: Long = 0
)
